### ISSUE TITLE
Add instructions for IDE

### ISSUE THREAD

**Issue by m-reza-rahman** [eclipse-ee4j/starter#116]
Add instructions on how to use Archetypes from an IDE such as Eclipse.

**Comment by nikitakoselev** [comment]
Is it still relevant?
I can add instructions for Eclipse and IntelliJ to the readme.md file, if that helps.

**Comment by m-reza-rahman** [comment]
Yes, very relevant. It's best to put the content in the Wiki I think. We can then prominently link it from the site and the GitHub readme.

**Comment by nikitakoselev** [comment]
I would like to concentrate on areas where my help is needed most.

**Comment by m-reza-rahman** [comment]
The UI is the highest priority. Do you think you could help deliver that? I estimate it will take about 50-70 hours of work.

This is lower priority by comparison.

**Comment by nikitakoselev** [comment]
I assume you refer to #171.
I will do #171 then, as it is higher priority ☺

**Comment by m-reza-rahman** [comment]
There is some work on this here: https://github.com/eclipse-ee4j/starter/pull/231
